### PR TITLE
Add a `make` function to use Pastel without ppx

### PR DESCRIPTION
**Pull request by tmattio**
This PR creates a function `create` as an alternative to the createElement function used by the JSX ppx.

This allows users who don't use the PPX (OCaml) to use Pastel with a nicer API.

In OCaml:
```
Pastel.create ~color:Pastel.Blue ["World"]
```
vs 
```
Pastel.createElement ~color:Pastel.Blue ~children:["World"] ()
```

### CODE REVIEW

**Comment by kyldvs** [comment]
Awesome! This will also help for using Pastel in places like: https://fb.workplace.com/groups/567987173671510/permalink/828579050945653/

No need to change yet, but I wonder if we use `Pastel.make` rather than `Pastel.create`. I think I see the former more often

**Comment by tmattio** [comment]
@kyldvs I like `make` better as well!

I wonder if it would make sense to pass it a string instead of a list of string also. The user would be responsible to concatenate the strings if needed.

**Comment by kyldvs** [comment]
Keeping it as a list might be nice so we can maintain the nested feature of pastel

```
Pastel.(make(
  ~color=Blue,
  [
    "Just Blue",
    make(~bold=true, ["Blue and bold"]),
    "Just Blue",
  ],
))
```

**Comment by tmattio** [comment]
@bandersongit Any thoughts on this? :)

File: src/pastel/Pastel.rei [hunk 1]
```diff
@@ -100,6 +100,25 @@ let magentaBright: decorator;
 let cyanBright: decorator;
 let whiteBright: decorator;
 
+/** Pastel.make allows non-pxx users to use Pastel.
+   * If you are using Reason, we recommend that you use the JSX ppx along side with Pastel.
+   * `<Pastel italic=true> "World" </Pastel>` is equivalent to `Pastel.make("World", ~italic=true)` */
```
[the remark this answers is not in the document]
Comment (tmattio) [reply src/pastel/Pastel.rei]: Done, thanks for the review!

**Pull request closed without merging**